lib/kennel/models/slo.rb

```diff
@@ -16,9 +17,15 @@ class Slo < Record
         tags: -> { @project.tags },
         query: -> { DEFAULTS.fetch(:query) },
         description: -> { DEFAULTS.fetch(:description) },
-        monitor_ids: -> { DEFAULTS.fetch(:monitor_ids) }
+        monitor_ids: -> { DEFAULTS.fetch(:monitor_ids) },
+        thresholds: -> { DEFAULTS.fetch(:thresholds) }
       )
 
+      def initialize(project, *args)
+        super(project, *args)
+        raise ValidationError, "Threshold warning must be greater-than critical value" if thresholds.any? { |t| t[:warning].to_f < t[:critical].to_f }
```

Comment:
message does not match check, needs to be `<=`
```suggestion
        raise ValidationError, "Threshold warning must be greater-than critical value" if thresholds.any? { |t| t[:warning].to_f <= t[:critical].to_f }
```